app/_src/kubernetes-ingress-controller/reference/combined-services-from-different-httproutes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Combined Services From Different HTTPRoutes 
+type: reference
+purpose: |
+  Introduce the feature gate to consolidate {{site.base_gateway}} services by combining rules from different HTTPRoutes
+alpha: true
+---
+
+As have been done to `Ingress`es, {{site.kic_product_name}} can consolidate rules from different `HTTPRoute`s with the same
+combination of backend services and translate them into one {{site.base_gateway}} service to reduce the number of {{site.base_gateway}} services.
+The feature is enabled when the feature gate `CombinedServicesFromDifferentHTTPRoutes` is set to `true`.
+When the feature is enabled, The rules with the same combination of backend services (combination of namespace, name, port and weight in `backendRefs` of rules)
+in all `HTTPRoute`s within the same namespace will be translated to one {{site.base_gateway}} service.
```

Comment:
I think it might be easier to explain first what happens without this feature gate. Then explain what happens with this feature gate to finally explain how to enable it.

Or maybe roughly like:

## What's this feature gate does?

Without this feature gate, each combination of route/service/backend ends up being a different gateway service.
When this feature gate is enabled combination of namespace, name, port and weight in `backendRefs` of rules 
across all `HTTPRoute`s within the same namespace will be translated to one {{site.base_gateway}} service.

## How to turn it on

The feature is enabled when the feature gate `CombinedServicesFromDifferentHTTPRoutes` is set to `true`.

## How translation is done ( was _Changes on Translated {{site.base_gateway}} Service Names_)

Reply:
Updated.